lambdas/service/app.py

```diff
@@ -185,37 +239,56 @@ def openapi():
 
 
 health_spec = {
-    'responses': {
-        '200': {
-            'description': '200 response',
-            'content': {
-                'application/json': {
-                    'schema': {'$ref': '#/components/schemas/Empty'}
-                }
-            }
-        }
-    },
     'tags': ['Health']
 }
 
 
 @app.route('/health', methods=['GET'], cors=True, method_spec={
     **health_spec,
-    'summary': 'List health information for webservice'
+    'description': 'List health information for Azul service.',
+    'summary': 'Check Azul service health',
+    'responses': {
+        '200': {
+            '$ref': '#/components/responses/OkHealth'
+        },
+        '503': {
+            '$ref': '#/components/responses/UnavailableHealth'
+        }
+    }
 })
 @app.route('/health/{keys}', methods=['GET'], cors=True, path_spec={
     'parameters': [
         {
             'name': 'keys',
             'in': 'path',
             'required': True,
```

Comment:
```suggestion
            'required': False,
```

Reply:
@danielsotirhos it seems like the specifications require this param to be se tot true, https://swagger.io/specification/#parameterObject.